src/components/Listbox/Listbox.vue

```diff
@@ -0,0 +1,323 @@
+<template lang="pug">
+div
+  template
+    KeypressListener(
+      keyEvent="keydown",
+      :keyCode="String(Key.DownArrow)",
+      :handler="handleDownArrow",
+    )
+    KeypressListener(
+      keyEvent="keydown",
+      :keyCode="String(Key.UpArrow)",
+      :handler="handleUpArrow",
+    )
+    KeypressListener(
+      keyEvent="keydown",
+      :keyCode="String(Key.Enter)",
+      :handler="handleEnter",
+    )
+  VisuallyHidden
+    div(aria-live="polite") {{ loading ? loading : null }}
+  ul(
+    v-if="$slots.default",
+    tabindex="0",
+    role="listbox",
+    :class="listboxClassName",
+    :aria-label="inCombobox ? undefined : accessibilityLabel",
+    :aria-labelledby="comboboxListboxContext.textFieldLabelId || ''",
+    :aria-busy="Boolean(loading)",
+    :id="listId",
+    @focus="handleFocus",
+    @blur="handleBlur",
+    ref="listboxRef",
+  )
+    slot
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import {
+  Component,
+  Prop,
+  Provide,
+  Inject,
+  Ref,
+  Watch,
+} from 'vue-property-decorator';
+import debounce from 'lodash/debounce';
+import { classNames } from 'polaris-react/src/utilities/css';
+import { scrollIntoView } from 'polaris-react/src/utilities/scroll-into-view';
+import { Key } from 'polaris-react/src/types';
+import type {
+  ListboxContextType, NavigableOption,
+} from 'polaris-react/src/utilities/listbox';
+import { closestParentMatch } from 'polaris-react/src/utilities/closest-parent-match';
+import {
+  listboxSectionDataSelector,
+} from 'polaris-react/src/components/Listbox/components/Section/selectors';
+import { ComboboxListboxType } from 'polaris-react/src/utilities/combobox/context';
+import styles from '@/classes/Listbox.json';
+import { useUniqueId } from '@/utilities/unique-id';
+import { KeypressListener } from '../KeypressListener';
+import { VisuallyHidden } from '../VisuallyHidden';
+
+export type ArrowKeys = 'up' | 'down';
+
+export const scrollable = {
+  props: { 'data-polaris-scrollable': true },
+  selector: '[data-polaris-scrollable]',
+};
+
+const LISTBOX_OPTION_SELECTOR = '[data-listbox-option]';
+const LISTBOX_OPTION_VALUE_ATTRIBUTE = 'data-listbox-option-value';
+
+const DATA_ATTRIBUTE = 'data-focused';
+const ARIA_ATTRIBUTE = 'aria-activedescendant';
+
+@Component({
+  components: {
+    KeypressListener,
+    VisuallyHidden,
+  },
+})
+export default class Listbox extends Vue {
+  @Inject({ default: {} }) comboboxListboxContext!: ComboboxListboxType;
```

Comment:
If you wanna use @Inject then you need it to be @Provide something. Right now you don't have it. It had to be @Provide somewhere. You need to check it to make sure it's gonna be provided and it has a key not just a default value like this. Try to use it properly! 
Check it out https://learnvue.co/2020/03/designing-vue3-plugins-using-provide-and-inject/#using-provide-and-inject-in-the-options-api